wordpress-mobile/WordPress-iOS
# Update gutenberg to 1.16.1

To test:
 - Smoke test the GB editor
 - Check that tapping on links is working correctly
 - Check that inserting new lines in multiline blocks ( quote, lists, image captions) works correctly

Update release notes:
